extensions/positron-python/python_files/positron/positron_ipykernel/ui.py

```diff
@@ -202,9 +203,19 @@ def open(self, url, new=0, autoraise=True):
                     ):
                         is_plot = True
                         break
-                # windows will not accept file:// at beginning of url
-                if os.name == "nt":
-                    url = urlparse(url).netloc or urlparse(url).path
+
+            # get path to the python_files/positron dir
+            parent = str(Path(__file__).parent.parent)
+
+            # bug where os.access is True on Windows when there is no access
+            # if the html file was not given a specific path, it will populate
+            # inside the positron_python extension. instead, put it in the cwd
```

Comment:
This makes sense and is probably a better UX for Positron overall!

This would have also been an issue for Workbench. I'm not sure that the `move` will solve that case e.g. one user could overwrite another user's file, but we could address that separately.

Reply:
I believe there are `move` settings to not overwrite a file, or we could add a timestamp like `plot-TIMEHASH.html` so each output is unique.

Reply:
The initial write by Bokeh will still be a problem. I'm not sure if that's configurable in Bokeh... It should always be a last resort, but we might have to patch something, like we do with Holoviews: https://github.com/posit-dev/positron/blob/f37f4f5044a2a619e73d5db61a31e37fbd3faf18/extensions/positron-python/python_files/positron/positron_ipykernel/holoviews.py#L9

Reply:
The option here would be to patch `bokeh.io.util._no_access` to always return True; I think we'd want to check for notebook mode, since that might not need it. The biggest difference between that approach and shuffling files around is that users would not see the html file be generated in their working directory on `show()`, but it would be generated if `output_file` is set. This patch seems relatively safe, since not having access is already valid/handled path in the bokeh library (just not triggered correctly).

If we have a collection of these patches, we might want them in a single directory for ease of discovery(?).